webapp/channels/src/components/admin_console/access_control/editors/cel_editor/editor.scss

```diff
@@ -0,0 +1,257 @@
+.cel-editor {
+    margin-bottom: 24px;
+
+    &__container {
+        position: relative;
+        display: flex;
+        overflow: auto;
+        height: auto;
+        flex-direction: column;
+        border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
+        border-radius: 4px;
+        margin-bottom: 8px;
+        background: var(--center-channel-bg);
+        overflow-y: auto;
+
+        .policy-editor-placeholder {
+            position: absolute;
+            z-index: 10;
+            top: 13px;
+            left: 30px;
+            color: rgba(0,0,0, 0.40);
+            font-family: monospace;
+            font-size: 12px;
+            pointer-events: none;
+            user-select: none;
+            white-space: pre-wrap;
+        }
+    }
+
+    &__cursor-position {
+        position: absolute;
+        right: 8px;
+        border-radius: 4px;
+        color: var(--button-color);
+        font-family: monospace;
+        font-size: 12px;
+        pointer-events: none;
+    }
+
+    &__input {
+        overflow: auto;
+        width: 100%;
+        height: auto;
+        min-height: 120px;
+        max-height: 600px;
+        //overflow-y: auto;
+        flex-grow: 0;
+        padding: 12px;
+        border: 1px solid var(--button-bg);
+        border-radius: 4px 4px 0 0;
+        font-family: monospace;
+        resize: vertical;
+        transition: border-color 0.15s ease;
+
+        &:focus {
+            outline: none;
+        }
+
+        &::placeholder {
+            color: rgba(var(--center-channel-color-rgb), 0.56);
+        }
+
+        .policyEditor {
+            outline: none;
+        }
+
+        .cel-editor__container[data-status-color='var(--error-text)'] & {
```

Comment:
(Post-merge) I commented about this elsewhere, but these should be set using CSS classes and not using data attributes